app/controllers/concerns/hyrax/admin/users_controller_behavior.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# OVERRIDE FILE from Hyrax v2.9.0
+module Hyrax
+  module Admin
+    module UsersControllerBehavior
+      extend ActiveSupport::Concern
+      include Blacklight::SearchContext
+      included do
+        with_themed_layout 'dashboard'
```

Comment:
Hyrax 3.x adds a `ensure_admin!` before action here. I think we are getting a better result by doing the authorize! :read, User below, but it should be noted and the "OVERRIDE line should be bumped up to hyrax 3